osu.Game.Tournament/Components/FumoBeatmapPanel.cs

```diff
@@ -302,6 +374,16 @@ private void runAnimation(ColourInfo? colour = null)
             // Stop any transform process (if exists) first
             FinishTransforms(true);
 
+            banPill.FinishTransforms(true);
+            banPill.Alpha = 0;
+            banPill.Y = 0;
+
+            trophyIcon.FinishTransforms(true);
+            trophyIcon.Alpha = 0;
+            trophyIcon.Y = 0;
```

Comment:
Can add a quick fade-out animation for the pill and icon to make the transform smoother.

```suggestion
            banPill.FinishTransforms(true);
            banPill.FadeOut(300, Easing.OutQuint);
            banPill.MoveToY(0, 300, Easing.OutQuint);

            trophyIcon.FinishTransforms(true);
            trophyIcon.FadeOut(300, Easing.OutQuint);
            trophyIcon.MoveToY(0, 300, Easing.OutQuint);
```

Reply:
Applied.